config.json

```diff
@@ -67,6 +66,19 @@
     },
     {
       "core": true,
+      "difficulty": 2,
+      "slug": "phone-number",
+      "topics": [
+        "control_flow_conditionals",
+        "control_flow_loops",
+        "dynamic",
+        "regular_expressions"
+      ],
+      "unlocked_by": "hamming",
```

Comment:
Move phone-number after bob. Set the `core` parameter to false. An! set the `unlocked_by` to `word-count`

I'm thinking this would be dependent on the Word Count exercise being merged in and then it will be the first non-core exercise to branch off.